### PR TITLE
featured various resources

### PR DESCRIPTION
**Pull request by SondraE**
## Cute animal pic
hedgehog

- [Trello card](https://trello.com/c/S7jDqAR2/54-i-want-a-few-tweaks-to-resources-list)
- [Preview](https://deploy-preview-35--oddleventy.netlify.com/)

## Description
- Most popular blog posts (according to Google Analytics) set to large feature.
- Taxonomy allows quickly featuring or showing images for entire post "types"
- Type "flags" show up in both list and page view
- Workshops and Talks show upcoming events, while other types show summary
- …unrelated, quote grids wait for more screen pace before going side-by-side

## Impacted areas of website
Resource list & detail pages

### CODE REVIEW

**Comment by mirisuzanne** [comment]
@SondraE Do you want a way to automate featuring particular tags/page-types, like workshops and talks? That seems better than manually updating all those posts.

Also: how should we be handling extra space from different sized cards? Space between like they are now, or stretch the cards vertically so they all match along a row?

**Comment by SondraE** [comment]
- Yes! I'd love for talks and workshops to always be feature: true, by default.
- For spacing, I love to see what it looks like when we add in images for video cards and text for article cards. I think I'm leaning toward stretching the cards vertically so they match along the bottom of the row. (There's a Trello card that's ready for Dev for the other Resources updates I'm referring to.)

**Comment by mirisuzanne** [comment]
[I want a few tweaks to Resources List](https://trello.com/c/S7jDqAR2/54-i-want-a-few-tweaks-to-resources-list)

**Comment by mirisuzanne** [comment]
@jgerigmeyer @stacyk I did some cleanup here, addressing review comments and also the tagLink bug that Stacy reported. Can  you take another look and give final sign-off?

File: content/blog/sitepoint/sass-book.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ sub: A preview of Jump Start Sass, chapter 11
 date: 2016-03-01
 canonical: https://www.sitepoint.com/architecture-in-sass/
 tags:
-  - Links
+  - Link
   - Sass
   - Open Source
   - Books
```
Comment (stacyk) [inline content/blog/sitepoint/sass-book.md]: If we are doing Links to Link, is there a reason not to change Books to Book?

File: src/scss/components/_post.scss [hunk 2]
```diff
@@ -86,100 +48,68 @@
...
-  grid-column: 1 / -1;
-  margin: var(--gutter) 0;
-}
+.type-flag {
+  @include tools.font-family('alt');
+  background: var(--highlight);
+  color: var(--bg);
+  font-size: var(--xsmall);
+  grid-area: type;
+  place-self: start end;
```
Comment (stacyk) [inline src/scss/components/_post.scss]: whoa, I haven't seen this yet. <googling it now>

**Comment by stacyk** [comment]
OH, and I should mention, tags work now as I expect they would :)

File: src/scss/components/_post.scss (hunk 2 above)
Comment (mirisuzanne) [reply src/scss/components/_post.scss]: `place-*` is the shorthand for `align-*` and `justify-*` 😄

**Pull request merged**